CMakeLists.txt

```diff
@@ -10,9 +10,8 @@ list(APPEND CMAKE_MODULE_PATH "${CIME_ROOT}/externals/pio2/cmake")
 
 list(APPEND CMAKE_MODULE_PATH ${CIME_CMAKE_MODULE_DIRECTORY})
 include(CIME_utils)
-find_package(NetCDF)
-include_directories(${NETCDF_INCLUDE_DIRS})
-
+find_package(NetCDF COMPONENTS C Fortran)
```

Comment:
I'm not sure if it was necessary to list both C and Fortran components. But it seems this could be needed on machines where the C and Fortran libraries are in different places. If you feel this change was wrong, I can try to revert it.

However, the change in the following line (include_directories) IS needed on my Mac: NETCDF_INCLUDE_DIRS was empty, but NetCDF_C_INCLUDE_DIRS and NetCDF_Fortran_INCLUDE_DIRS are non-empty (they are the same on my Mac but - tied in with the above comment - may not be the same on other machines?)